languages/python2/config.json

```diff
@@ -1,21 +1,7 @@
 {
-    "build": {
-        "env_variables": [
-            {"name":"PATH", "value":"/opt/anaconda2/bin:$PATH"}
-        ],
-        "files": [],
-        "artifacts": [
-            {"source":"/home/algo/.local", "destination":"/home/algo/.local/"},
-            {"source":"/opt/algorithm/src", "destination":"/opt/algorithm/src/"},
-            {"source":"/opt/algorithm/algorithmia.conf", "destination":"/opt/algorithm/algorithmia.conf"}
-        ]
-    },
-    "runtime": {
-        "env_variables": [
-            {"name":"PATH", "value":"/opt/anaconda2/bin:$PATH"}
-        ],
-        "files": [
-            {"source":"bin/pipe", "destination":"/opt/algorithm/bin/pipe"}
-        ]
-    }
+    "artifacts": [
+        {"source":"/home/algo/.local", "destination":"/home/algo/.local/"},
+        {"source":"/opt/algorithm/src", "destination":"/opt/algorithm/src/"},
+        {"source":"/opt/algorithm/algorithmia.conf", "destination":"/opt/algorithm/algorithmia.conf"}
```

Comment:
From a debug-ability standpoint, it would be nice to also copy over the requirements file

Also, it seems like templates might have other dirs that are siblings to src. Can we just take all of /opt/algorithm?

Reply:
hmm... I guess I hadnt looked at any python templates yet (and seems weird  they'd have things other than src but idk). I think copying the entire directory might have clobbered the pipe script that is already in the runner image... maybe if the destination has a trailing / or something it would work but it's been like 5 months since this was made so I cant recall.

Reply:
yeah, that would clobber the bin/pipe script - I do think the requirements file would be nice to have. 

I feel like this is fine for now and am not sure if we want to think about a way to let people customize it in the future

Reply:
Arguably all of the stuff living at the root of templates should be published as packages. 

That said, doesn't change the fact that there might be other things at the root that should be copied

Reply:
i added in the requirements.txt - think that's a good idea.  i'm not sure if we'll need to customize it in the future, but when we actually have templates to  look at we can reconsider this.  alternatively, the templates might have a config file indicating all the things to copy too? Idk